Tests/ApolloCodegenTests/CodeGeneration/FileGenerators/SchemaModuleFileGeneratorTests.swift

```diff
@@ -4,23 +4,23 @@ import ApolloCodegenTestSupport
 import Nimble
 
 class SchemaModuleFileGeneratorTests: XCTestCase {
+  let rootURL = URL(fileURLWithPath: CodegenTestHelper.outputFolderURL().path)
```

Comment:
Ignore this too, shouldn't have been in this PR. Subsequent PR, based on this one, focuses on the file generators.